src/sockperf.cpp

```diff
@@ -165,7 +165,7 @@ static const AOPT_DESC common_opt_desc[] = {
       "Show the help message and exit." },
     { OPT_TCP,                AOPT_NOARG,                       aopt_set_literal(0),
       aopt_set_string("tcp"), "Use TCP protocol (default UDP)." },
-    { 'i', AOPT_ARG, aopt_set_literal('i'), aopt_set_string("ip"), "Listen on/send to ip <ip>." },
+    { 'i', AOPT_ARG, aopt_set_literal('i'), aopt_set_string("addr", "ip"), "Listen on/send to ip <ip>, or address <name>" },
```

Comment:
please consider
`Listen on/send to address in IPv4, IPv6, UNIX domain socket format`